man/systemd-vmspawn.xml

```diff
@@ -117,6 +117,21 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--qemu-swtpm=</option><replaceable>BOOL</replaceable></term>
```

Comment:
why prefix this switch name with "qemu" btw?

I'd like to abstract the user virtualizer a bit, hence where things aren't inherently qemu specific I'd give this a generic name.

moreover, there are multiple vtpm implementations areound, swtpm is just one, I'd also abstract this in the name a bit, and thus name this switch with a generic "vtpm", which gives us some liberty to swap out what is backed by.

Reply:
As with many of these I chose the name because that is what mkosi calls the argument.
I'll aim to make more generic names in future.
Fixed in 37e75950ff.